src/Sylius/Bundle/CoreBundle/Sitemap/Builder/SitemapBuilder.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+ 
+namespace Sylius\Bundle\CoreBundle\Sitemap\Builder;
+
+use Sylius\Bundle\CoreBundle\Sitemap\Factory\SitemapFactoryInterface;
+use Sylius\Bundle\CoreBundle\Sitemap\Provider\UrlProviderInterface;
+use Symfony\Component\DependencyInjection\ContainerAware;
+
+/**
+ * @author Arkadiusz Krakowiak <arkadiusz.krakowiak@lakion.com>
+ */
+class SitemapBuilder implements SitemapBuilderInterface
+{
+    /**
+     * @var SitemapFactoryInterface
+     */
+    private $sitemapFactory;
+
+    /**
+     * @var array
+     */
+    private $providers;
+
+    /**
+     * @param SitemapFactoryInterface $sitemapFactory
+     */
+    public function __construct(SitemapFactoryInterface $sitemapFactory)
+    {
+        $this->sitemapFactory = $sitemapFactory;
+        $this->providers = [];
```

Comment:
Should be removed in favor of https://github.com/Sylius/Sylius/pull/3642/files#r54566641